ts/defs.bzl

```diff
@@ -136,7 +136,21 @@ def ts_project(
         data: Files needed at runtime by binaries or tests that transitively depend on this target.
             See https://bazel.build/reference/be/common-definitions#typical-attributes
 
-        deps: List of labels of other rules that produce TypeScript typings (.d.ts files)
+        deps: List of labels of other rules that produce TypeScript typings (`.d.ts` files)
+
+            If this list contains linked npm packages, npm package store targets or other targets that provide
+            `JsInfo`, `NpmPackageStoreInfo` providers are gathered from `JsInfo`. This is done directly from
+            the `npm_package_store_deps` field of these. For linked npm package targets, the underlying
```

Comment:
I don't think people will know what `NpmPackageStoreInfo` or `npm_package_store_deps` is?

Reply:
I think that's ok for now. The SoT for this docstring snippet is in rules_js. We can improve on it there first.